platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 5.10.0-alpha.1
+Starting this release, the Mapbox Maps SDK for iOS will use a pre-built GL Native binary licensed under the [Mapbox Terms of Service](https://www.mapbox.com/legal/tos). The license of the Maps SDK for iOS remains BSD-2. For more information on using this and future releases, see [README.md](https://github.com/mapbox/mapbox-gl-native-iOS/blob/master/README.md). For more details on licensing, see [LICENSE.md](https://github.com/mapbox/mapbox-gl-native-iOS/blob/master/LICENSE.md).
+
+### ✨ New features
+
+- Source Modularization.
+  Introduces a SourceManager + *SourceFactory classes similar to the work done on layer modularization.
+
+- [core] Enable source "volatile" flag initialization in style.
+
+  The source `volatile` flag value can be defined in the style as following:
+
+  ```
+  "example_source": {
+    "volatile": true
+  }
+  ```
+
+  For the sources with the `volatile` flag set, the tiles are not stored in the local storage. The `volatile` flag is applicable for all source types, which allow loading tiles from a remote server i.e. `vector`, `raster`, `raster-dem` source types. For other source types this setting is ignored.
+
+- [core] Enable `Renderer::collectPlacedSymbolData()` API for all map modes.
```

Comment:
What's the impact of this change?

Reply:
If this applies to iOS developers, we should rephrase and reformat to match the rest of our changelog entries.